homeassistant/components/rest_command.py

```diff
@@ -72,6 +75,11 @@ def async_register_rest_command(name, command_config):
             template_payload = command_config[CONF_PAYLOAD]
             template_payload.hass = hass
 
+        headers = None
+        if CONF_CONTENT_TYPE in command_config:
+            content_type = command_config[CONF_CONTENT_TYPE]
+            headers = {HTTP_HEADER_CONTENT_TYPE: content_type}
```

Comment:
Please use: https://github.com/aio-libs/aiohttp/blob/master/aiohttp/hdrs.py#L45 so we are sure that we right expose it to api